File: CMI/Contract/Order/StatusHistory.cs
using System;

namespace CMI.Contract.Order
{
    public class StatusHistory
    {
        public int Id { get; set; }
        public int OrderItemId { get; set; }
        public DateTime StatusChangeDate { get; set; }
        public OrderStatesInternal FromStatus { get; set; }
        public OrderStatesInternal ToStatus { get; set; }
        public string ChangedBy { get; set; }
    }
}

File: CMI/Web.Clients/web-management/src/app/modules/ordermanagement/model/statusHistory.ts
import {InternalStatus} from '@cmi/viaduc-web-core';

export class StatusHistory {
	public id: number;
	public orderItemId: number;
	public statusChangeDate: Date;
	public fromStatus: InternalStatus;
	public toStatus: InternalStatus;
	public changedBy: string;
}
